lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, app, nworkers, **kwargs):
             k8s_cleanup_job=dict(map=str, valid=lambda s: s in {"onsuccess", "always", "never"}, default="always"),
             k8s_pod_retries=dict(map=int, valid=lambda x: int(x) >= 0, default=3),
             k8s_walltime_limit=dict(map=int, valid=lambda x: int(x) >= 0, default=172800),
-            k8s_unschedulable_walltime_limit=dict(map=int, valid=lambda x: int(x) >= 0, default=1800),
+            k8s_unschedulable_walltime_limit=dict(map=int, valid=lambda x: int(x) >= 0, default=18000),
```

Comment:
5 hours still seems short. 5 days feels like a better number. For example, on a number of TACC resources, larger jobs may sit in the queue for a couple of days.

Reply:
Have just dropped that commit for now. Will issue a new PR with the walltime fix. We should probably make the `k8s_unschedulable_walltime_limit` an optional parameter, defaulting to unlimited.